Makefile.ibi

```diff
@@ -121,11 +146,51 @@ $(IBI_CLUSTER_CONFIG_PATH): ibi-certs credentials/pull-secret.json
 $(IBI_CLUSTER_CONFIG_MANIFESTS):
 	mkdir -p $(IBI_CLUSTER_CONFIG_MANIFESTS)
 
+.PHONY: build-openshift-install
+build-openshift-install: openshift-installer ## Build openshift-install binary
+	cd openshift-installer && ./hack/build.sh
+
+.PHONY: ibi-install-config
+ibi-install-config:
+	$(info Creating IBI install-config.yaml)
+	@mkdir -p $(IBI_WORK_DIR)
+	@# CLUSTER_NETWORK and CLUSTER_SVC_NETWORK are ommited because they are not used (the ones from the seed image are used)
+	@INSTALLER_WORKDIR=../$(IBI_WORK_DIR) \
+	PULL_SECRET='$(PULL_SECRET)' \
+	SSH_KEY_PUB_PATH=$(SSH_KEY_PUB_PATH) \
+	INSTALLATION_DISK=$(IBI_INSTALLATION_DISK) \
+	CLUSTER_NAME=$(IBI_CLUSTER_NAME) \
+	BASE_DOMAIN=$(IBI_DOMAIN) \
+	MACHINE_NETWORK=$(IBI_VM_NETWORK) \
+		make -C $(SNO_DIR) ../$(IBI_WORK_DIR)/install-config.yaml
+
+.PHONY: ibi-imagebased-config
+ibi-imagebased-config:
+	$(info Creating IBI imagebased-config.yaml)
+	@mkdir -p $(IBI_WORK_DIR)
+	@< $(IMAGEBASED_CONFIG_TEMPLATE) \
+		VM_NAME=$(IBI_VM_NAME) \
+		HOST_IP=$(IBI_VM_IP) \
+		HOST_MAC=$(IBI_VM_MAC) \
+		HOST_ROUTE=$(shell $(virsh) net-dumpxml $(IBI_NET_NAME) | grep '<ip ' | xargs -n1 | grep address | cut -d = -f 2) \
+		RELEASE_REGISTRY=$(IBI_RELEASE_REGISTRY) \
+		envsubst > $(IBI_WORK_DIR)/imagebased-config.yaml
+
+.PHONY: imagebasedconfig.iso
+imagebasedconfig.iso: RELEASE_VERSION=$(SEED_VERSION)
+imagebasedconfig.iso: $(IBI_WORK_DIR)/imagebasedconfig.iso ## Create imagebasedconfig.iso using openshift-installer
+
+.PHONY: $(IBI_WORK_DIR)/imagebasedconfig.iso
+$(IBI_WORK_DIR)/imagebasedconfig.iso: RELEASE_VERSION=$(SEED_VERSION)
```

Comment:
Consider replacing the `sudo cp $@ $(LIBVIRT_IMAGE_PATH)/ibi-config.iso` with updating `ibi-attach-config.iso` to use `$(IBI_WORK_DIR)/imagebasedconfig.iso`